Add missing changelog entries for ML (#3894)
elastic/docs-content: release-notes/elastic-cloud-serverless/index.md

* Speeds up field simulation in Streams [#241313]({{kib-pull}}241313)
* Updates action response codes [#240420]({{kib-pull}}240420)
* Fixes an infinite loop bug in the **Investigation guide** editor [#240472]({{kib-pull}}240472)
* [ML] Perform query field validation for rerank task type [#137219](https://github.com/elastic/elasticsearch/pull/137219)
* Preserve deployments with zero allocations during assignment planning [#137244](https://github.com/elastic/elasticsearch/pull/137244)

> * [ML] Skip dataframes when disabled [#137220](https://github.com/elastic/elasticsearch/pull/137220)

Reviewer: ```suggestion
* Skips dataframes when disabled [#137220](https://github.com/elastic/elasticsearch/pull/137220)
```

* Adds triple-quote support to the Manual Ingest Pipeline Processor editor [#236595]({{kib-pull}}236595)
* Introduces the German locale for Kibana in `beta` [#236903]({{kib-pull}}236903)
* Adds an advanced option to disable filtering of file-backed volumes and CD-ROMs in the **Device Control** plugin [#236620]({{kib-pull}}236620)

> * Adds RerankRequestChunker [#130485](https://github.com/elastic/elasticsearch/pull/130485)

Reviewer: ```suggestion
* Adds `RerankRequestChunker` [#130485](https://github.com/elastic/elasticsearch/pull/130485)
```

* Fixes layout issues with the **Parse in streams** button on smaller flyouts [#236548]({{kib-pull}}236548)
* Displays `(missing value)` and `(empty)` instead of `null` in charts and tables [#233369]({{kib-pull}}233369)
* Fixes privilege requirements for reindexing indices in Upgrade Assistant [#237055]({{kib-pull}}237055)
* Reset health status on successful noop [#135653](https://github.com/elastic/elasticsearch/pull/135653)

> * Switches `TextExpansionQueryBuilder` and TextEmbeddingQueryVectorBuilder to return 400 instead of 500 errors [#135800](https://github.com/elastic/elasticsearch/pull/135800)

Reviewer: ```suggestion
* Switches `TextExpansionQueryBuilder` and `TextEmbeddingQueryVectorBuilder` to return 400 instead of 500 errors [#135800](https://github.com/elastic/elasticsearch/pull/135800)
```

* Adds support for package spec v3.5 [#235942]({{kib-pull}}235942)
* Adds in-product documentation for the {{esql}} `FORK` command [#236494]({{kib-pull}}236494)
* Adds **View in discover** button in alert details page for SLO burn rate and ES query rules [#233855]({{kib-pull}}233855)

> * Adds custom headers support for openai text embeddings [#134960](https://github.com/elastic/elasticsearch/pull/134960)

Reviewer: ```suggestion
* Adds custom headers support for OpenAI text embeddings [#134960](https://github.com/elastic/elasticsearch/pull/134960)
```

* Adds `time` field to the get data views response schema [#235975]({{kib-pull}}235975)
* Adds `managed` field to the get data views response schema [#236237]({{kib-pull}}236237)
* Validates {{ls}} pipeline IDs sent to Kibana APIs [#236347]({{kib-pull}}236347)

> * [ML] Add .reindexed-v7-ml-anomalies-* to anomaly results template index pattern (#135270) [#135286](https://github.com/elastic/elasticsearch/pull/135286)

Reviewer: ```suggestion
* Adds `.reindexed-v7-ml-anomalies-*` to anomaly results template index pattern (#135270) [#135286](https://github.com/elastic/elasticsearch/pull/135286)
```

* Tolerates mixed types in datafeed stats sort [#135096](https://github.com/elastic/elasticsearch/pull/135096)

> * Fixes Missing Config Errors [#134963](https://github.com/elastic/elasticsearch/pull/134963)

Reviewer: ```suggestion
* Fixes a bug in the GET _transform API that incorrectly claims some transform configurations are missing [#134963](https://github.com/elastic/elasticsearch/pull/134963)
```

> * Gracefully shutdown model deployment when node is removed from assignment routing [#134673](https://github.com/elastic/elasticsearch/pull/134673)

Reviewer: ```suggestion
* Gracefully shuts down model deployment when node is removed from assignment routing [#134673](https://github.com/elastic/elasticsearch/pull/134673)
```

* Adds search functionality to the Query rules details page [#232579]({{kib-pull}}232579)
* Adds a link to Agent Builder in the **View Data** dropdown [#234679]({{kib-pull}}234679)
* Adds the AutoOps Search tier page, which provides project-level insights and deeper insights into {{serverless-short}} resources (VCUs) and performances

> 

Reviewer: Remove blank line
```suggestion
```

> * Adding headers support for OpenAI chat completion [#134504](https://github.com/elastic/elasticsearch/pull/134504)

Reviewer: ```suggestion
* Adds headers support for OpenAI chat completion [#134504](https://github.com/elastic/elasticsearch/pull/134504)
```

* Fixes an Anomaly Detection bug where custom URLs omitted generated fields in datafeed preview requests [#234709]({{kib-pull}}234709)
* Ensures full tool traces are displayed in flyouts [#234654]({{kib-pull}}234654)

* Returns 429 status when `RequestExecutorService` queue full [#134178](https://github.com/elastic/elasticsearch/pull/134178)
* Fixes model assignment error handling and assignment explanation generation [#133916](https://github.com/elastic/elasticsearch/pull/133916)

> * Implements latency improvements for `EIS` integration [#133861](https://github.com/elastic/elasticsearch/pull/133861)

Reviewer: ```suggestion
* Implements latency improvements for EIS integration [#133861](https://github.com/elastic/elasticsearch/pull/133861)
```

* Removes the default port the from interactive setup cluster address form, unless specified [#230582]({{kib-pull}}230582)
* Fixes positioning of the **Add rule** popover on the Role Mappings page [#231551]({{kib-pull}}231551)

> * Updates `EIS` sparse and dense embedding max batch size to 16 [#132646](https://github.com/elastic/elasticsearch/pull/132646)

Reviewer: ```suggestion
* Updates EIS sparse and dense embedding max batch size to 16 [#132646](https://github.com/elastic/elasticsearch/pull/132646)
```

* Fixes a bug that prevented saving linked TSVB visualizations when changing the data view [#228685]({{kib-pull}}228685)
* Fixes a null property error in the Elasticsearch Serverless Playground [#230729]({{kib-pull}}230729)

> * Inference API disable partial search results [#132362](https://github.com/elastic/elasticsearch/pull/132362)

Reviewer: ```suggestion
* Disables inference API partial search results [#132362](https://github.com/elastic/elasticsearch/pull/132362)
```

* Implements navigation UI for the **Overview Page** in **Entity Analytics** [#221748]({{kib-pull}}221748)
* Adds support for partial result handling in **ES|QL** [#223198]({{kib-pull}}223198)
* Adds an **Executable Name** tab to the TopN view [#224291]({{kib-pull}}224291)

> * SageMaker Elastic Payload [#129413](https://github.com/elastic/elasticsearch/pull/129413)

Reviewer: ```suggestion
* Adds a SageMaker Elastic payload [#129413](https://github.com/elastic/elasticsearch/pull/129413)
```

* Fixes an issue with **Anomaly Explorer** where the selected Overall swimlane bucket is not respected for `viewBy jobId` in Machine Learning [#222845]({{kib-pull}}222845).
* Fixes error handling when one or more connectors is deleted [#221958]({{kib-pull}}221958).

> * Fixes `NPE` when `date_buckets` aggregation is missing in the response [#128974](https://github.com/elastic/elasticsearch/pull/128974)

Reviewer: ```suggestion
* Fixes null pointer exception when `date_buckets` aggregation is missing in the response [#128974](https://github.com/elastic/elasticsearch/pull/128974)
```

* Updates the "Highlighted fields" button in the details flyout and enables the feature flag in {{sec-serverless}} [#221862]({{kib-pull}}221862)
* Introduces new `empty` states for the Change Point Detection page in {{ml-cap}} [#219072]({{kib-pull}}219072)

* Adding VoyageAI's v3.5 models [#128241](https://github.com/elastic/elasticsearch/pull/128241)
* Integrates OpenAi Chat Completion in SageMaker [#127767](https://github.com/elastic/elasticsearch/pull/127767)
* Improves exception for trained model deployment scale up timeout [#128218](https://github.com/elastic/elasticsearch/pull/128218)
* InferenceService support aliases [#128584](https://github.com/elastic/elasticsearch/pull/128584)
* Implemented ChatCompletion task for Google VertexAI with Gemini Models [#128105](https://github.com/elastic/elasticsearch/pull/128105)

> * Adding configurable inference service [#127939](https://github.com/elastic/elasticsearch/pull/127939)

Reviewer: ```suggestion
* Adds configurable inference service [#127939](https://github.com/elastic/elasticsearch/pull/127939)
```

* Reworks cookie and session storage to prevent unexpected logouts for certain users with certain use cases [#220430]({{kib-pull}}220430)
* Changes the AI Connector description [#221154]({{kib-pull}}221154)

> * Checks alias during update [#124825](https://github.com/elastic/elasticsearch/pull/124825)

Reviewer: ```suggestion
* Checks the transform update index against its alias during updates [#124825](https://github.com/elastic/elasticsearch/pull/124825)
```

> * Use internal user for internal inference action [#128327](https://github.com/elastic/elasticsearch/pull/128327)

Reviewer: ```suggestion
* Uses internal user for internal inference action [#128327](https://github.com/elastic/elasticsearch/pull/128327)
```

* Fixes incorrect validation for names containing asterisks in **ES|QL** [#219832]({{kib-pull}}219832)
* Fixes overridden SSL config in full agent policy advanced YAML for Fleet [#219902]({{kib-pull}}219902)

* Appends all data to Chat Completion buffer [#127658](https://github.com/elastic/elasticsearch/pull/127658)

> * Adding timeout to request for creating inference endpoint [#126805](https://github.com/elastic/elasticsearch/pull/126805)

Reviewer: ```suggestion
* Adds timeout to request for creating inference endpoint [#126805](https://github.com/elastic/elasticsearch/pull/126805)
```

* Removes metrics and logs from the `get_service_stats` API [#218346](https://github.com/elastic/kibana/pull/218346)
* Allows you to customize the table tab [#218686](https://github.com/elastic/kibana/pull/218686)
* Enables keyboard navigation for the create annotations form [#217918](https://github.com/elastic/kibana/pull/217918)

> * {{esql}} random sampling [#125570](https://github.com/elastic/elasticsearch/pull/125570)

Reviewer: ```suggestion
* Adds {{esql}} random sampling [#125570](https://github.com/elastic/elasticsearch/pull/125570)
```

* Fixes title announcements in the details step of the anomaly detection job wizard [#218570](https://github.com/elastic/kibana/pull/218570)
* Fixes incorrect optimization for endpoint artifacts [#216437](https://github.com/elastic/kibana/pull/216437)

> * Adding missing onFailure call for Inference API start model request [#126930](https://github.com/elastic/elasticsearch/pull/126930)

Reviewer: ```suggestion
* Adds missing onFailure call for inference API start model request [#126930](https://github.com/elastic/elasticsearch/pull/126930)
```

* Reuses shared integration policies when duplicating agent policies in Fleet [#217872]({{kib-pull}}217872)
* Enables adding badges to all list items in the side navigation except the section header [#217301]({{kib-pull}}217301)

* Adding endpoint creation validation to ElasticsearchInternalService [#123044](https://github.com/elastic/elasticsearch/pull/123044)

> * Bedrock Cohere Task Settings Support [#126493](https://github.com/elastic/elasticsearch/pull/126493)

Reviewer: ```suggestion
* Adds support for Amazon Bedrock Cohere task settings [#126493](https://github.com/elastic/elasticsearch/pull/126493)
```

* Filters inference connectors that lack existing endpoints in **Connectors** [#217641]({{kib-pull}}217641)
* Fixes focusability and keyboard access issues with the **Export** tab in the **Share this dashboard** modal [#217313]({{kib-pull}}217313)

* Fixes ELAND endpoints not updating dimensions [#126537](https://github.com/elastic/elasticsearch/pull/126537)

> * Fixes `ModelLoaderUtils`.split() to pass tests [#126009](https://github.com/elastic/elasticsearch/pull/126009)

Reviewer: ```suggestion
* Changes `ModelLoaderUtils.split` to return the correct number of chunks and ranges [#126009](https://github.com/elastic/elasticsearch/pull/126009)
```

* Adds the ability to limit notes per document instead of globally in Elastic Security Serverless [#214922]({{kib-pull}}214922)
* Adds the ability to add badges to subitems in the side navigation [#214854]({{kib-pull}}214854)

> * [ML] Check if the anomaly results index has been rolled over (#125404) [#125786](https://github.com/elastic/elasticsearch/pull/125786)

Reviewer: ```suggestion
* Checks if the anomaly results index has been rolled over (#125404) [#125786](https://github.com/elastic/elasticsearch/pull/125786)
```

* Adds a custom link color option for the top banner [#214241]({{kib-pull}}214241)
* Updates the task state version after execution [#215559]({{kib-pull}}215559)

> * Returns a Conflict status code if the model deployment is stopped by a user [#125204](https://github.com/elastic/elasticsearch/pull/125204)

Reviewer: ```suggestion
* Returns a conflict status code if the model deployment is stopped by a user [#125204](https://github.com/elastic/elasticsearch/pull/125204)
```

* Changes "Close project" to "Log out" in nav menu in serverless mode [#211463]({{kib-pull}}211463)
* Fixes search profiler index reset field when query is changed [#215420]({{kib-pull}}215420)

> * Fixes `AlibabaCloudSearchCompletionAction` not accepting ChatCompletionInputs [#125023](https://github.com/elastic/elasticsearch/pull/125023)

Reviewer: ```suggestion
* Fixes `AlibabaCloudSearchCompletionAction` not accepting `ChatCompletionInputs` [#125023](https://github.com/elastic/elasticsearch/pull/125023)
```

* Introduces GA support for the [`semantic_text`](elasticsearch://reference/elasticsearch/mapping-reference/semantic-text.md) field type on {{serverless-full}}
* Adds the ability for users to [customize prebuilt rules](https://github.com/elastic/kibana/issues/174168). Users can modify most rule parameters, export and import prebuilt rules — including customized ones — and upgrade prebuilt rules while retaining customization settings [#212761]({{kib-pull}}212761)

* Propagates product use case http header to `EIS` [#124025](https://github.com/elastic/elasticsearch/pull/124025)
* Integrates with DeepSeek API [#122218](https://github.com/elastic/elasticsearch/pull/122218)

> * Adds `max.chunks` to `EmbeddingRequestChunker` to prevent `OOM` [#123150](https://github.com/elastic/elasticsearch/pull/123150)

Reviewer: ```suggestion
* Adds `max.chunks` to `EmbeddingRequestChunker` to prevent out of memory error [#123150](https://github.com/elastic/elasticsearch/pull/123150)
```

* Upgrades AWS v2 SDK to 2.30.38 [#124738](https://github.com/elastic/elasticsearch/pull/124738)

> * Exposes ``input_type`` option at root level for `text_embedding` task type in Perform Inference API [#122638](https://github.com/elastic/elasticsearch/pull/122638)

Reviewer: ```suggestion
* Exposes `input_type` option at root level for `text_embedding` task type in the perform Inference API [#122638](https://github.com/elastic/elasticsearch/pull/122638)
```

* Fixes appearance of warnings in the {{esql}} editor [#213685]({{kib-pull}}213685)
* Makes the Apply time range switch visible in the Job selection flyout when opened from the Anomaly Explorer [#213382]({{kib-pull}}213382)

* Migrates `model_version` to `model_id` when parsing persistent elser inference endpoints [#124769](https://github.com/elastic/elasticsearch/pull/124769)

> * Avoids unnecessary calls to Task#getDescription in model download [#124527](https://github.com/elastic/elasticsearch/pull/124527)

Reviewer: ```suggestion
* Avoids unnecessary calls to `Task#getDescription` in model download [#124527](https://github.com/elastic/elasticsearch/pull/124527)
```
